packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -381,14 +401,15 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                 data-variant="light"
                 data-on-change="$ctrl.onSelectedModeChange(mode)"
                 data-required
+                data-disabled="mode.disabled"
             >
                 <oui-select-picker-section>
                     <p data-translate="{{:: mode.description1}}"></p>
                     <p data-translate="{{:: mode.description2}}"></p>
                 </oui-select-picker-section>
             </oui-select-picker>
         </div>
-        <div>
+        <div data-ng-show="false">
```

Comment:
What's the utility ?
```suggestion
        <div>
```

Reply:
Its already been take cared in next PR: https://github.com/ovh/manager/pull/10523